sql/core/src/test/resources/sql-functions/output.out

```diff
@@ -0,0 +1,3409 @@
+
+-- Function name: rollup
+-- !query
+SELECT name, age, count(*) FROM VALUES (2, 'Alice'), (5, 'Bob') people(age, name) GROUP BY rollup(name, age)
+-- !query schema
+struct<name:string,age:int,count(1):bigint>
+
+
+-- Class name: org.apache.spark.sql.catalyst.expressions.LessThanOrEqual
+
+-- Function name: <=
+-- !query
+SELECT 2 <= 2
+-- !query schema
+struct<(2 <= 2):boolean>
+-- !query
+SELECT 1.0 <= '1'
+-- !query schema
+struct<(CAST(1.0 AS DOUBLE) <= CAST(1 AS DOUBLE)):boolean>
+-- !query
+SELECT to_date('2009-07-30 04:17:52') <= to_date('2009-07-30 04:17:52')
+-- !query schema
+struct<(to_date('2009-07-30 04:17:52') <= to_date('2009-07-30 04:17:52')):boolean>
+-- !query
+SELECT to_date('2009-07-30 04:17:52') <= to_date('2009-08-01 04:17:52')
+-- !query schema
+struct<(to_date('2009-07-30 04:17:52') <= to_date('2009-08-01 04:17:52')):boolean>
+-- !query
+SELECT 1 <= NULL
+-- !query schema
+struct<(1 <= CAST(NULL AS INT)):boolean>
+
+
+-- Class name: org.apache.spark.sql.catalyst.expressions.Elt
+
+-- Function name: elt
+-- !query
+SELECT elt(1, 'scala', 'java')
+-- !query schema
+struct<elt(1, scala, java):string>
+
+
+-- Class name: org.apache.spark.sql.catalyst.expressions.aggregate.Skewness
+
+-- Function name: skewness
+-- !query
+SELECT skewness(col) FROM VALUES (-10), (-20), (100), (1000) AS tab(col)
+-- !query schema
+struct<skewness(CAST(col AS DOUBLE)):double>
+-- !query
+SELECT skewness(col) FROM VALUES (-1000), (-100), (10), (20) AS tab(col)
+-- !query schema
+struct<skewness(CAST(col AS DOUBLE)):double>
+
+
+-- Class name: org.apache.spark.sql.catalyst.expressions.Left
+
+-- Function name: left
+-- !query
+SELECT left('Spark SQL', 3)
+-- !query schema
+struct<left('Spark SQL', 3):string>
+
+
+-- Class name: org.apache.spark.sql.catalyst.expressions.StringTrimRight
+
+-- Function name: rtrim
+-- !query
+SELECT rtrim('    SparkSQL   ')
+-- !query schema
+struct<rtrim(    SparkSQL   ):string>
+
+
+-- Class name: org.apache.spark.sql.catalyst.expressions.Lead
+
+-- Function name: lead
+
+
+-- Class name: org.apache.spark.sql.catalyst.expressions.ArrayForAll
+
+-- Function name: forall
+-- !query
+SELECT forall(array(1, 2, 3), x -> x % 2 == 0)
+-- !query schema
+struct<forall(array(1, 2, 3), lambdafunction(((namedlambdavariable() % 2) = 0), namedlambdavariable())):boolean>
+-- !query
+SELECT forall(array(2, 4, 8), x -> x % 2 == 0)
+-- !query schema
+struct<forall(array(2, 4, 8), lambdafunction(((namedlambdavariable() % 2) = 0), namedlambdavariable())):boolean>
+-- !query
+SELECT forall(array(1, null, 3), x -> x % 2 == 0)
+-- !query schema
+struct<forall(array(1, CAST(NULL AS INT), 3), lambdafunction(((namedlambdavariable() % 2) = 0), namedlambdavariable())):boolean>
+-- !query
+SELECT forall(array(2, null, 8), x -> x % 2 == 0)
+-- !query schema
+struct<forall(array(2, CAST(NULL AS INT), 8), lambdafunction(((namedlambdavariable() % 2) = 0), namedlambdavariable())):boolean>
```

Comment:
Actually, we do not need all of them. How about just showing the first one?

Reply:
OK. we only need the first one. Because other SQL and schema no more value. Thanks for your remind.